cmd/scality-cosi-driver/cmd.go

```diff
@@ -65,5 +88,12 @@ func run(ctx context.Context) error {
 		return fmt.Errorf("failed to start the provisioner server: %w", err)
 	}
 
-	return server.Run(ctx)
+	err = server.Run(ctx, registry)
+	shutdownCtx, shutdownCancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer shutdownCancel()
```

Comment:
```suggestion
	shutdownCtx, _ := context.WithTimeout(context.Background(), 5*time.Second)
```

I don't think it adds anything to retrieve the cancel function just to defer it, the context will be canceled in 5 seconds anyway. That is unless your linter is not letting you do that.

Reply:
It makes sense to change it. 
Thanks for catching that.

Reply:
Changed in https://github.com/scality/cosi-driver/pull/83/commits/35a2063fe8892ed7b8fdf10f91b1c2aca2f9a848